<html><body><pre>
yarn run v1.9.4
$ eslint .

/home/jenkins/workspace/org_ui-plugin-find-contact_PR-23/project/findContact/FindContactContainer.js
  153:16  error  Missing trailing comma  comma-dangle

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.

info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
</pre><body></html>

--- a/findContact/FindContactContainer.js
+++ b/findContact/FindContactContainer.js
@@ -150,7 +150,7 @@ class FindContactContainer extends React.Component {
       source: this.source,
       visibleColumns,
       data: {
-        records
+        records,
       },
     });
   }

--- a/package.json
+++ b/package.json
@@ -14,6 +14,7 @@
     "actsAs": ["plugin"],
     "pluginType": "find-contact",
     "okapiInterfaces": {
+      "organizations-storage.categories": "1.0",
       "organizations-storage.contacts": "2.0"
     },
     "displayName": "ui-plugin-find-contact.meta.pluginTitle"

